homeassistant/brands/august.json

```diff
@@ -0,0 +1,5 @@
+{
+  "domain": "august",
+  "name": "August Home",
+  "integrations": ["august", "yalexs_ble"]
```

Comment:
Shouildn't yalexs be put in the Yale brand ?

Reply:
`yalexs_ble` also works with August locks.
Assa Abloy (Yale’s parent company) purchased August in 2017.
New devices use the Yale Access branding, however, August locks work as well 🤷